content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to animals
+
+# Rebuttal
+
+## Quick Response
+
+> Superiority supposes an arbitrary line that identifies one group as deservedly superior to another. Common examples are that men are superior to women (sexism) and whites are superior to all other ethnicities (racism).  
```

Comment:
can we change 
`whites are superior to all other ethnicities`
to 
`one ethnicity is superior to all others`
As the first one potentially diminishes the significance of other forms of racism that happen across the globe. 

I don't _think_ we need to do the same with sexism because I can't think of any examples of any other direction than men oppressing women. Other than possibly transphobia, so that might be good to mention for balance if you like!